samples/wavlib/src/main/cpp/io/stream/FileInputStream.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <unistd.h>
+
+#include "FileInputStream.h"
+
+namespace wavlib {
+
+long FileInputStream::read(void *buff, long numBytes) {

```

Comment:
A long can change size in different builds. How about int32_t or int64_t?

Reply:
Yes. It's a trade off between portability and simplicity. int32_t and int64_t aren't defined anywhere in this project.
Let's talk about it.